### PR TITLE
Added tests for Label

### LINKED ISSUE #945: Add tests for Label

### PR DESCRIPTION
**Pull request by vysakh-m**
Fixes #945

### CODE REVIEW

**Comment by vysakh-m** [comment]
@amaldinesh7 _a Please review

File: tests/Label.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import Label from "../lib/components/Label";
+import userEvent from "@testing-library/user-event";
+
+describe("Label", () => {
+  it("should render without error", () => {
+    const { getByText } = render(
+      <Label>
+        <p>Content</p>
```
Comment (amaldinesh7) [inline tests/Label.test.js:10]: we don't need to have a `p` tag inside render

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@vysakh-m _a Please check the comment and assign to @ajmaln once fixed.

**Comment by vysakh-m** [comment]
Resolved suggestion. @ajmaln _a Please review.

**Review by ajmaln — COMMENTED** [review]
@vysakh-m _a Please check the suggestion

**Comment by vysakh-m** [comment]
@ajmaln _a Resolved suggestion. Please review

File: lib/components/Label.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ const Label = ({
             })}
             onClick={onClick}
           >
-            <HelpIcon size={16} {...otherHelpIconProps} />
+            <HelpIcon data-testid="icon" size={16} {...otherHelpIconProps} />
```
Comment (ajmaln) [inline lib/components/Label.js:39]: let's pass the `data-testid` in `helpIconProps`

**Review by ajmaln — APPROVED** [review]

**Pull request merged**